proof_test.go

```diff
@@ -914,6 +914,33 @@ func TestProofVerificationWithPostState(t *testing.T) {
 
 			proof, _, _, _, _ := MakeVerkleMultiProof(root, postroot, data.keystoprove, nil)
 
+		keys:
```

Comment:
I'm adding a bit more of checking here. I want to check that the generated pre-state and post-state values match what was originally created in the proofed trees.